src/app/ApplicationTemplate.Windows/ApplicationTemplate.Windows.csproj

```diff
@@ -86,7 +86,7 @@
 		<PackageReference Include="Reactive.Annex.Uno.WinUI" Version="2.0.0" />
 		<PackageReference Include="ReviewService.NativePrompters" Version="1.0.0" />
 		<PackageReference Include="Uno.Core.Extensions.Logging.Singleton" Version="4.1.1" />
-		<PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
+		<PackageReference Include="Serilog.Sinks.File" Version="6.0.0" />
```

Comment:
This seems unrelated?

Reply:
I saw the downgrade in a previous PR, I can make the change in a different commit